sormas-backend/src/main/java/de/symeda/sormas/backend/util/RightsAllowed.java

```diff
@@ -0,0 +1,19 @@
+package de.symeda.sormas.backend.util;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Documented
+@Retention(RUNTIME)
+@Target({TYPE, METHOD})
+public @interface RightsAllowed {
```

Comment:
Missing comment that this class mimics `@RolesAllowed` to avoid performance problemns